libs/platform/src/lib/form/form-group/form-group-header/form-group-header.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { FieldGroup } from '../../form-helpers';
+import { HintOptions } from '@fundamental-ngx/platform/shared';
+
+@Component({
+    // eslint-disable-next-line @angular-eslint/component-selector
+    selector: '[fdp-form-group-header]',
+    templateUrl: './form-group-header.component.html',
+    styles: [
+        `
+            h1.fd-form-group__header-text {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+            }
```

Comment:
Maybe move it to fund-styles if it's not Angular specific.

Reply:
No, header did not have these specs